spec/acceptance/custom_log_from_organizer_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+describe "Log from an organizer with a custom logger" do
+  context "when overriding the global LightService organizer" do
+    let(:global_logger_organizer) do
+      Class.new do
+        extend LightService::Organizer
+
+        def self.call(number)
+          with(:number => number).reduce(actions)
+        end
+
+        def self.actions
+          [
+            TestDoubles::AddsOneAction,
+            TestDoubles::AddsTwoAction,
+            TestDoubles::AddsThreeAction
+          ]
+        end
+      end
+    end
+
+    let(:custom_logger_string) { StringIO.new }
+    let(:custom_logger_organizer) do
+      custom_logger = Logger.new(custom_logger_string)
+
+      Class.new do
+        extend LightService::Organizer
+        log_with custom_logger
+
+        def self.call(coffee, this_hot = :very_hot)
+          with(:milk => this_hot, :coffee => coffee)
+            .reduce(TestDoubles::MakesLatteAction,
+                    TestDoubles::AddsTwoActionWithFetch)
+        end
+      end
+    end
+
+    it "logs in own logger" do
+      global_logger_organizer.call(1)
+      custom_logger_organizer.call(:coffee => "Cappucino")
+
+      expect(custom_logger_string.string).to include("MakesLatteAction")
+      expect(custom_logger_string.string).to_not include("AddsOneAction")
+    end
+  end
+end
```

Comment:
This test is great! 👍 

Can you add one more test case, where the default logger goes into one sink (StringIO in thise case), and a custom logger goes into another one? I'd like to make sure it's backward compatible

Reply:
Of course! Done. :+1: